contracts/partial/yToken/lendingMethods.ligo

```diff
@@ -634,13 +599,8 @@ function accrueInterest(
     then failwith("yToken/interest-update-wrong-state");
```

Comment:
In the PR, some of the condition statements are moved to the beginning of the function. This one can also be moved.

Reply:
It actually can't since it requires `token` to be retrieved.